core/platform/taskqueue/cloud_taskqueue_services.py

```diff
@@ -29,8 +29,6 @@
 from google.protobuf import timestamp_pb2
 
 CLIENT = tasks_v2.CloudTasksClient()
-# This is needed so that we can see whether the create_task is retried.
-retry._LOGGER.setLevel(logging.DEBUG)  # pylint: disable=protected-access
```

Comment:
Just to check, what is the effect of removing this? 

If multiple retries fail, don't we want to have the logs? (Or is this reported in some other way as a hard failure?)

Reply:
If entities fail multiple types or with an error that is not repeated it will be reported. This change is just to get rid of logs that were used for debugging if the failures (not caused by our code but by the API) were repeated (which they were).

Reply:
OK, sg, thanks for explaining!